### DISCUSSION TITLE
Add option for "Status Bar"

### DISCUSSION THREAD

**Discussion by decembre** [onemen/TabMixPlus#281] [Ideas]
It should be great to have an option inside "TabMixPlus" to add a Status Bar,
And control its height / icons size,  Background color.

Previously it was devoted to an userscript "Status Bar" in Chrome folder of the Firefox's Profile folder, 
but we can't fine tuning it easily...

**Comment by onemen** [comment]
What will be the content of the `Status Bar`?

**Reply by decembre** [reply]
Found in <a href="https://github.com/xiaoxiaoflood/firefox-scripts">userChromeJS - firefox-scripts [xiaoxiaoflood in GhitHub]</a>:
_"Status Bar"
Brings back the good old status bar (also known as Addon Bar) at the bottom, with status text plus any buttons you want._ 

That's the Old Tool bar i had in Waterfox:
Named too " "Add-on Bar""
At the bottom of the windows, an extra toolbar to put in it some addon icons.
![Status-4-Evar-Customize](https://github.com/onemen/TabMixPlus/assets/868778/36432597-b2bf-48bb-930e-c4887cbb52b8)


Some links:
<a href="https://www.optimiced.com/en/2018/02/11/restore-status-bar-in-firefox-quantum/">Where is my status bar in Firefox Quantum? Here’s a way to restore it!</a>

**Comment by onemen** [comment]
I am not going to add this script to Tab Mix Plus.

**Comment by decembre** [comment]
Ok, but can you add your own  option to it ?

And i don't understand how/why <a href="https://github.com/xiaoxiaoflood/firefox-scripts">userChromeJS - firefox-scripts [xiaoxiaoflood in GhitHub]</a> is / was related to your project:
They are some similar file they use (by example Fx-folder) and i don't know if they are compatible with your.

**Comment by onemen** [comment]
> Ok, but can you add your own option to it ?
`Status Bar` options in Tab Mix Plus, where?

Since Firefox removed the internal component that loads legacy extension, in order to install Tab Mix Plus, or any other legacy extension, , you have to install 3rd party helper scripts.

The same comment apply to user scripts, `Status Bar` is part of large user scripts collection.

**Comment by pyrates999** [comment]
The only thing the status bar can be used for now is showing the loading text at the bottom.  Otherwise it is not really used I don't think.
